### PR TITLE
createenclave management api

### PR DESCRIPTION
**Pull request by bvavala**
partially addresses #358

### CODE REVIEW

File: fabric/bin/peer.sh [hunk 1]
```diff
@@ -376,8 +376,28 @@ handle_lifecycle_chaincode_commit() {
...
+        *)
+        die "createenclave: invalid option"
+        ;;
+        esac
+    done
+
+    # - createenclave can only be run on FPC chaincodes
+    FILES_NUM=$(ls -1 ${FABRIC_STATE_DIR}/is-fpc-c-chaincode.${CC_NAME}.* 2> /dev/null | wc -l) 
+    if [ ${FILES_NUM} -eq 0 ]; then
+        die "createenclave: $CC_NAME does not have fpc-c type"
```
Comment (g2flyer) [inline fabric/bin/peer.sh:399]: Minor rewording suggestion as peer cli uses language (--lang) to refer to different types
```suggestion
        die "createenclave: $CC_NAME is not written in language 'fpc-c'"
```

**Comment by mbrandenburger** [comment]
Thanks for finishing this one!

**Comment by mbrandenburger** [comment]
Seems updating hello world example was missing. I created #389 to add this.

**Comment by g2flyer** [comment]
it's actually in my PR ...

**Pull request merged**